### PR TITLE
feat: add Hungarian locale

### PR DESCRIPTION
**Pull request by szabolcstarnai**
Added Hungarian locale for (some) animals, commerce, date, finance, internet, name, phone_number and word

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -82,4 +82,8 @@ REVISION
 .project
 .tasks-cache
 *DONOTVERSION*
+<<<<<<< HEAD
```
Comment (damienwebdev) [inline .gitignore:85]: You committed a merge conflict.

File: package.json [hunk 2]
```diff
@@ -44,13 +44,13 @@
   "devDependencies": {
     "browserify": "^16.5.2",
     "coveralls": "^3.1.0",
+    "eslint": "^6.5.1",
```
Comment (damienwebdev) [inline package.json:47]: Remove this diff.

File: lib/locales/hu/animal/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+var animal = {};
+module['exports'] = animal;
+animal.dog = require("./dog");
+animal.cat = require("./cat");
+// animal.snake = require("./snake");
```
Comment (damienwebdev) [inline lib/locales/hu/animal/index.js:5]: Don't leave comments.

**Review by damienwebdev — DISMISSED** [review]
We will need someone who speaks Hungarian (other than the author) to review this prior to merging.

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
Just in case, please revert `examples/browser/js/faker.js` and `examples/browser/js/faker.min.js`
We will keep track of that on our side

**Comment by Shinigami92** [comment]
We now have all locales written in TypeScript.
Please have a look and rewrite your files like we have done with all other files.

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
We now have all locales written in TypeScript.
Please have a look and rewrite your files like we have done with all other files.

**Comment by Shinigami92** [comment]
@patak-dev I hope I didn't miss-assume your language :sweat_smile: 
But could you have a quick look about this?

That would be really nice

**Comment by patak-dev** [comment]
I'm Argentinian @Shinigami92, not the best for the task 😅

**Comment by Shinigami92** [comment]
> I'm Argentinian @Shinigami92, not the best for the task sweat_smile

Damnd, okay, sorry about that. Hope it's not to cold in the snow :snowflake: :snowboarder: 
Take care :wave:

**Review by szabolcstarnai — COMMENTED** [review]
In Hungarian 'prepositions' don't really exist, practically we only use postpositions.

**Review by Shinigami92 — DISMISSED** [review]
We need someone that can approve this PR in terms of Hungarian

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]
Looks good format wise.

**Comment by Shinigami92** [comment]
If someone else uses these, the new locales and detect that something is wrong, they can raise a new issue / open new PR

**Pull request merged**